src/json_dsl/validators/at_least_one_of.rs

```diff
@@ -9,7 +9,7 @@ pub struct AtLeastOneOf {
 impl AtLeastOneOf {
     pub fn new(params: &[&str]) -> AtLeastOneOf {
         AtLeastOneOf {
-            params: params.iter().map(|s| s.to_string()).collect(),
+            params: params.iter().map(|s| (*s).to_string()).collect(),
```

Comment:
```suggestion
            params: params.iter().map(|s| s.to_string()).collect(),
```

Reply:
nope, that’s not the same and doesn’t work (which I don’t understand, either)

Reply:
ah, wrong context: this one was actually changed by clippy (it does that since around 1.39 or so)